shared/chat/conversation/messages/special-bottom-message.js

```diff
@@ -46,6 +47,7 @@ const mapStateToProps = (state: TypedState, ownProps: OwnProps) => {
       : null
 
   return {
+    measure: ownProps.measure,
```

Comment:
this missing plumbing was what caused the red skull box to overlap sometimes (and in this case never appear!)